### ISSUE TITLE
Alter the config to pass a `--quiet-deps` flag

### ISSUE THREAD

**Issue by nimmolo** [tablecheck/dartsass-sprockets#25]
Our app has a ton of SCSS deprecation warnings. 

Is there a way to pass the dart-sass build flag to quiet these?
Something like how `dartsass-rails` has `Rails.application.config.dartsass.build_options << " --quiet-deps"`

**Comment by nimmolo** [comment]
Actually, groping in the dark here... I checked out the way `config_options` is passed in `Sprockets::SassCompressor` and `SassTemplate` so I tried just passing the arg like this, and it ~seems to have worked~. 

```ruby 
# config/production.rb 
  config.sass.quiet_deps = true
```

~Would you be open to a PR to document this?~
Update, it does not work to pass the config. The deprecation warnings are reported unpredictably on compile (as far as I can tell), and I just thought it had silenced them. The gem actually doesn't handle these args, so i'm working on a PR to handle them.

**Comment by johnnyshields** [comment]
I'm happy to accept a PR for this.

**Comment by nimmolo** [comment]
Thanks @johnnyshields - I realized later my approach did not work; it's only on the build that the deprecations get flagged (of course...) and I hadn't noticed that i wasn't triggering new builds. 😂  

Long story short, my flag was ignored. I'm not sure there is a way to pass that flag, but I will look into it again.

**Comment by nimmolo** [comment]
I'm going to try to enable a config like this.
```ruby
  # config/production.rb  
  config.sass.quiet_deps = true
```

It seems the config options for this gem are set in three different places though, and I'm a little confused by that.
I assume this is for compatibility with SassC's API.

Should I focus on one of these instead of the others?
- `/lib/sassc/rails/compressor.rb:9`
- `/lib/sassc/rails/railtie.rb:9`
- `/lib/sassc/rails/template.rb:26`

**Comment by jukra** [comment]
@nimmolo would you have a chance to test my PR https://github.com/tablecheck/dartsass-sprockets/pull/27? I think the changes are not needed for `compressor.rb` as it's compressing the css output, not compiling to it.
